### DISCUSSION TITLE
Login to Meteor web app via React-Native webview

### DISCUSSION THREAD

**Discussion by allenfuller** [TheRealNate/react-native-meteor#50] [Q&A] [answered]
Hi folks,

Curious if anyone else has run into this scenario:

We have an existing, large Meteor web app with a responsive frontend, but are ready to create a native mobile app. However, we would like to implement features gradually, so there will be cases where we want to render the current web app within the native mobile app.

Ergo, I'd like to handle login via the native app, but still be able to automagically login users to the web app where a web view is needed.

It seems like [the right path](https://forums.meteor.com/t/login-user-via-url-token/48584/3) is to use Meteor's `loginWithToken` method, but just wanted to double check and see if anyone else has tried that before.

Specifically, how would we generate a token in the native app that could be used to login to the web app?

Will keep digging, but seemed like it might be relevant to others. Grateful for this awesome package!

**Comment by not-ai-yet** [comment, marked as answer]
Hu @allenfuller, 

You can retrieve the React Native app's Meteor login token using `Meteor.getData(). _tokenIdSaved`.

If you pass this token to the webview you should be able to use it with `Meteor.loginWithToken`.

**Reply by allenfuller** [reply]
Awesome -- thanks Nate!
